src/client/table_client.rs

```diff
@@ -1986,6 +2003,15 @@ impl ObTableClientQueryImpl {
             }
         }
 
+        // defense for multiple partition aggreagtion
+        if self.aggregation_check() && partition_table.len() > 1 {
```

Comment:
I think we should make it more common, ``aggregation_check()`` is a part of **query_check**, and we could check others in the future.

Reply:
change the order

Reply:
add comment